test/inductor/test_aot_inductor.py

```diff
@@ -3649,12 +3649,8 @@ def fail_non_abi_compatible_cuda(is_skip=False):
     "test_dynamic_scalar": fail_minimal_arrayref_interface(is_skip=True),
     # https://github.com/pytorch/pytorch/issues/122980
     "test_fft_c2c": fail_stack_allocation(is_skip=True),
-    # TODO: test_freezing_abi_compatible_cpu fails,
-    #   AssertionError: None, i.e. optional output is not supported
-    "test_freezing": fail_with_and_without_stack_allocation(is_skip=True),
-    # TODO: test_linear_freezing_abi_compatible_cpu fails,
-    #   AssertionError: None, i.e. optional output is not supported
-    "test_linear_freezing": fail_with_and_without_stack_allocation(is_skip=True),
+    "test_freezing": fail_stack_allocation(is_skip=True),
+    "test_linear_freezing": fail_stack_allocation(is_skip=True),
```

Comment:
Being curious, what's the current error that prevents us from enabling these two tests?

Reply:
I changed to `fail_minimal_arrayref_interface`, which should be fine to ignore for now.